kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/ImageServiceBase.java

```diff
@@ -135,6 +136,10 @@ public String getActiveProcessImage(String containerId, long procInstId, String
             Collection<NodeInstanceDesc> activeLogs = dataService.getProcessInstanceHistoryActive(procInstId, qc);
             Collection<NodeInstanceDesc> completedLogs = dataService.getProcessInstanceHistoryCompleted(procInstId, qc);
             Collection<NodeInstanceDesc> fullLogs = dataService.getProcessInstanceFullHistory(procInstId, qc);
+
+            List<String> keyList = fullLogs.stream().collect(Collectors.groupingBy(NodeInstanceDesc::getNodeId, Collectors.counting())).entrySet().stream().filter(e -> e.getValue() == 1).map(Map.Entry::getKey).collect(Collectors.toList());
+            List<String> errors = fullLogs.stream().filter(nodeInstanceDesc -> (((org.jbpm.kie.services.impl.model.NodeInstanceDesc) nodeInstanceDesc).getType() == 6) && keyList.contains(nodeInstanceDesc.getNodeId())).map(NodeInstanceDesc::getNodeId).collect(Collectors.toList());
```

Comment:
@bxf12315 Type = 6 is not corresponding with the nodes in error https://github.com/kiegroup/droolsjbpm-knowledge/blob/7.x/kie-api/src/main/java/org/kie/api/runtime/manager/audit/NodeInstanceLog.java#L64